### DISCUSSION TITLE
Over-reaction to Road to rome blocked, intended behavior?

### DISCUSSION THREAD

**Discussion by capsicumw** [Keriew/augustus#996] [General]
On several occasions, the game thinks some new construction has blocked the road to Rome and deletes the recent construction. I remember this from the original game but as I remember it was only the last item constructed. 
With Augustus it doesn't just delete the recently constructed thing that blocks the road, it deletes every aqueduct and wall on the map along with small temples that have vacant land or road on all four sides(so in no way blocking anything. Some stuff years old, not recent constructions. Is this intended behavior?

**Comment by Keriew** [comment]
Yeah, the engineers trying to clear the road to Rome can be overzealous. It's partially complicated because the game doesn't really know what caused the blockage for various technical reasons. It would be possible to adjust this, but it's fundamentally a bit tricky (and the undo feature caused its own large share of issues). Still, getting the road to Rome blocked is a major issue for players.

I've adjusted the behaviour since the last release, so in the unstable and in the upcoming release, the game pauses, gives player a warning and then gives a chance to unblock the city before trashing the player's city. I think this should fix the problem in most use cases.

**Reply by capsicumw** [reply]
The pause and warning sounds like a good fix.
